languages/elixir/exercises/concept/booleans/test/rules_test.exs

```diff
@@ -1,57 +1,71 @@
 defmodule RulesTest do
   use ExUnit.Case
 
-  test "if ghost gets eaten" do
-    assert Rules.eat_ghost?(true, true)
-  end
+  describe "eat_ghost?/2" do
+    # @tag :pending
```

Comment:
Is there any use for this comment? I can't imagine people re-enabling this :)

```suggestion
    # @tag :pending
```

Reply:
Just to show that it can be, it's a common pattern in the elixir practice exercises.

Reply:
Okay! Makes sense.